### ISSUE TITLE
inventory data preparation workflow

### ISSUE THREAD

**Issue by cpiponiot** [Bioforest-project/inventories#6]

**Comment by GeraldineDerroire** [comment]
@cpiponiot I've seen that you have specify echo==TRUE for all chunck of the main quarto file. Do you really meant that? 
What I add in mine with that file is to provide a user friendly diagnostic of the data. If there are too many code chunck, I'm not sure it will help engage the PI, and it breaks the flow of the written text. I have changed it, but please confirm that you are happy with this (and we the way I have started to write the "writtent text " in the quarto document. Thanks !

**Comment by GeraldineDerroire** [comment]
How do we want to user to specify the dataset they use ? Do we want to ask them to call them a generic name (like "MyData.csv") and call that file?
At the moment, I have put the name of a specific dataset I use when I load the data, but this will need to be changed.

**Comment by mithilaunkule** [comment]
> How do we want to user to specify the dataset they use ? Do we want to ask them to call them a generic name (like "MyData.csv") and call that file? At the moment, I have put the name of a specific dataset I use when I load the data, but this will need to be changed.

we could ask them to name it "harmonized_dataset_sitename", and put it in the same folder as the code. In the code we can import files containing "harmonized_data".

**Comment by GeraldineDerroire** [comment]
> > How do we want to user to specify the dataset they use ? Do we want to ask them to call them a generic name (like "MyData.csv") and call that file? At the moment, I have put the name of a specific dataset I use when I load the data, but this will need to be changed.
> 
> we could ask them to name it "harmonized_dataset_sitename", and put it in the same folder as the code. In the code we can import files containing "harmonized_data". What do you think about this idea?

Yes, that is a possibility. But if a PI is in charge of several sites, they will put them all in the folder and this will create problem. But we can possibely ask them to put them one after the other. Or we have a first command in the markdown that prompt them to type the name of the site they want to work on. I've never done it in a markdown, but I think it should be too complicated. @cpiponiot what do you think?
